docs/Discussions

```diff
@@ -0,0 +1,12 @@
+## DevOps Mutualization SIG Discussions
+
+DevOps Mutualization SIG uses Github Discussions that are only visible to members who are part of the DevOps SIG [team](https://github.com/orgs/finos/teams/devops-mutualization-participants/) as agreed upon in the [meeting on Feb 18th 2020](https://github.com/finos/devops-mutualization/issues/18)
+Discussions augments  Issues in this repository for public discussions. 
+
+## How to Participate in DevOps SIG Discussions
+- If you are not in the FINOS Github org, email help@finos.org and request that your Github account is added.
```

Comment:
`If you are not in the FINOS Github org` **should be** `If you're are not a member of the [FINOS Github Organisation](https://github.com/orgs/finos/people)`

Reply:
`request that your Github account is added` **should be** `request an invitation to join from the FINOS team.`